Payload/bleemsync/etc/bleemsync/FUNC/0000_shared.funcs

```diff
@@ -5,16 +5,17 @@
 ###############################################################################
 
 pretty_logo_func(){ #ASCII PORN
-	bin=$bleemsync_path/etc/bleemsync/SUP/scripts/show_logo
-	chmod +x $bin && $bin
+	bin="$bleemsync_path/etc/bleemsync/SUP/scripts/show_logo"
+	chmod +x "$bin" && "$bin"
 }
 
-red_led_flash () {
+red_led_flash() {
   SLEEP=1
-  echo 0 > /sys/class/leds/green/brightness
   while true; do
+    echo 0 > /sys/class/leds/green/brightness
```

Comment:
`power_manage` sometimes updates the LEDs, so make sure you're actually displaying what you want.